source/includes/_08-deployment.md

```diff
@@ -32,6 +32,16 @@ It will take a while, but you should have your project deployed to your cloud pr
 
 If you make changes to your code, you can run `boost deploy` again to update your project in the cloud.
 
+### Introducing booster full-stack applications
+
+Booster now uploads your frontend application to the cloud too! Just follow these steps:
+
+- Create a `public` directory in the root of your booster project
```

Comment:
Right now what we're deploying is the folder `public/dist`. Check out this comment https://github.com/theam/booster/pull/87/files#r427215068

Reply:
I think we could change it to deploy `public`. I added a response to that comment telling this

Reply:
I'm changing it to deploy the content of `/public`